clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -1407,14 +1407,61 @@ check_filter_endpoint()
     return true;
```

Comment:
Title should start with "i#NNN: " if there is an associated issue: so "i#6187: Add ...".

Reply:
done.

Reply:
nit: There should be a colon after the number.

Reply:
I've updated the title accordingly.